cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -634,3 +636,76 @@ def _revert():
 
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.tier(2)
+@pytest.mark.parametrize(
+    "verbosity",
+    [
+        "1 (Verbose)",
+        "2 (More Verbose)",
+        "3 (Debug)",
+        "4 (Connection Debug)",
+        "5 (WinRM Debug)",
+        "0 (Normal)",
+    ],
+)
+@pytest.mark.meta(automates=[BZ(1460788)])
+def test_service_ansible_verbosity(
+    appliance,
+    request,
+    ansible_catalog_item,
+    ansible_service_catalog,
+    ansible_service_request,
+    ansible_service,
+    verbosity,
+):
+    """Check if the different Verbosity levels can be applied to service and
+    monitor the std out
+
+    Bugzilla:
+        1460788
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    pattern = '"verbosity"=>{}'.format(verbosity[0])
+    with update(ansible_catalog_item):
+        ansible_catalog_item.provisioning = {"verbosity": verbosity}
+        ansible_catalog_item.retirement = {"verbosity": verbosity}
+    # Log Validator
+    log = LogValidator(
+        "/var/www/miq/vmdb/log/evm.log", matched_patterns=[pattern]
+    )
+    # Start Log check or given pattern
+    log.fix_before_start()
+
+    @request.addfinalizer
```

Comment:
So, a finalizer is going to be called after each test execution is finished. So in this case, just because it is parametrized, it is still going to run everytime because each parametrization is a separate test. 

**Required** I think the best bet here is to create a module scoped fixtures that reads the default setting in the CFME gui, yield, and sets it back after all tests have been executed. That way, this is only run once.

Reply:
Not sure what you are telling, got confused here, where should I have finalizer there or not .?

Reply:
Considered the Verbosity `0 (Normal)`  as a default and setting back after every test execution in the finalizer.

Reply:
So two part problem here:

1) We are assuming Verbosity 0 is the default, I don't think we should make that assumption. What if it is 0 right now, but it is changed to 3 in the future. We should read what the default is set to and use that to rest.

2) So for the finalizer. Every time this test is executed, the finalizer will run. But how it is currently written, it is not setting back to 0. It is setting it to whatever the verbosity is parametrized for this instance of the test. What really needs to happen, is to read what the default is set to and use that in the finalizer. But, it makes no sense to do that every since parameterization of this run. So what I am suggesting, if create a module scope fixture that:
- reads the default and saves that to a variable
- yields
- Set the default back after all iterations of the test are run.

Reply:
Okay Fixture which will read and reset it in finalizer. Got it.

Reply:
@jawatts I have thought on this as all above procedure is part of code and not being used anywhere in the code, so keeping it in the test would give more impact.

Also, shuffled order of `Verbosity`, which will at the end add default `Verbosity` i.e. `"0 (Normal)"`. Which will act as `Finalizer`.

Reply:
@sbulage We are still assuming that `0` is the default and will always be the default. I don't think we should make the assumption. We should read what it is in the GUI first and reset it to that. Also, if the test fails at any point before `0`, that parameterization will not run. Also how the finalizer is written now, it is going to run every single test execution. So this finalizer is running 6 times. I don't think that is needed, that is why I suggested the fixture.

Reply:
Yeah, `Finalizer` will run after every test `parameter`, that's why I have removed that code from `Finalizer`. Updating PR now.